konduit-serving-pipeline/src/test/java/ai/konduit/serving/pipeline/impl/data/DataTest.java

```diff
@@ -112,12 +125,117 @@ public void testEmbeddedData() {
         Data layeredContainer = JData.singleton("upperLevel", ndContainer);
     }
 
-    @Ignore
     @Test
     public void testSerde() {
         Data someData = JData.singleton(KEY, Long.valueOf(200));
-        someData.save(new File("temp"));
-        Data restoredData = JData.fromFile(new File("temp"));
+        try {
+            someData.save(new File("temp"));
+        } catch (IOException e) {
+            log.error("IOException in DataTest.testSerde()", e);
+        }
+        Data restoredData = null;
+        try {
+            restoredData = JData.fromFile(new File("temp"));
+        } catch (IOException e) {
+            log.error("IOException in DataTest.testSerde()", e);
+        }
+        assertEquals(someData.get(KEY), restoredData.get(KEY));
+    }
+
+    @Test
+    public void testConvertToBytes() {
+        JData someData = (JData) JData.singleton(KEY, Long.valueOf(200));
```

Comment:
`Data someData = Data.singleton(KEY, 200L);`
Thought let's also handle the Integer -> Long internally too for this method

Reply:
Shouldn't we add int32 and float data types? We are dealing with Object ref and upcast is not possible.

Reply:
Let's do Float -> Double, and Integer -> Long
upcast is definitely possible, in 2 ways:
Use Number.longValue() and similar methods (all boxed primites are Number objects - Double, Float, Integer, Long, etc)
Cast to primitive:
double d = (Float)obj;

Reply:
Fixed with upcast.